docs/onboarding/8_16/operator/README.md

```diff
@@ -76,7 +76,7 @@ $ kubectl create namespace opentelemetry-operator-system
 ```
 $ helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
 $ helm repo update
-$ helm install --namespace opentelemetry-operator-system opentelemetry-kube-stack open-telemetry/opentelemetry-kube-stack --values ./resources/kubernetes/operator/helm/values.yaml --version 0.2.2
+$ helm upgrade --install --namespace opentelemetry-operator-system opentelemetry-kube-stack open-telemetry/opentelemetry-kube-stack --values ./resources/kubernetes/operator/helm/values.yaml --version 0.2.2
```

Comment:
Leftover?

Reply:
If we need to add that command for updates, which I think is helpful to include, 
then just add it as an addtional note

Reply:
I prefer 1 command instead of additional `If you need` in documentation, since `upgrade --install` is doing the same thing:
```
-i, --install                                    if a release by this name doesn't already exist, run an install
```
but not strong opinion, so added it as an addition - https://github.com/elastic/opentelemetry/pull/24/commits/9592941528f8ace151d1934a34c8a8633fbcbe27

Reply:
Ah nice did not know that tbh :) Had the impression that install is needed first time. Even better then

Reply:
Same for me, `upgrade --install` looks good.

Reply:
should I then revert https://github.com/elastic/opentelemetry/commit/9592941528f8ace151d1934a34c8a8633fbcbe27 and keep `helm upgrade --install` only? ot keep as is now?

Reply:
https://github.com/elastic/opentelemetry/pull/24/commits/8565bf8f1258ffd98f0ed112081d8ee03896f901